_targets.R

```diff
@@ -267,20 +272,36 @@ list(
   ),
   
   ##generate table of data to download from sciencebase
-  tar_target(p1_make_sb_dl_table,
-             make_dl_table(idtostart, outdir = "./1_fetch/out"),
+  tar_target(p1_sb_table_full,
```

Comment:
For file targets, the naming convention is to add the extension to the target name. So `p1_sb_table_full_csv` for this target. Can you change the other file target names, too?

Reply:
Addressed.